service/importer/service_file_test.go

```diff
@@ -21,60 +20,34 @@ func TestReadServiceFileDoesNotExist(t *testing.T) {
 	require.True(t, (len(data) == 0))
 }
 
-// Test validateServiceFileSchema function
-
-func TestValidateServiceFileSchema(t *testing.T) {
-	data, _ := readServiceFile("./tests/service-valid")
-	var body interface{}
-	_ = yaml.Unmarshal(data, &body)
-	body = convert(body)
-	result, err := validateServiceFileSchema(body, "service/importer/assets/schema.json")
-	require.NoError(t, err)
-	require.True(t, result.Valid())
-}
-
-func TestValidateServiceFileSchemaNotExisting(t *testing.T) {
-	data, _ := readServiceFile("./tests/service-valid")
-	var body interface{}
-	_ = yaml.Unmarshal(data, &body)
-	body = convert(body)
-	_, err := validateServiceFileSchema(body, "service/assets/not_existing")
-	require.Error(t, err)
-}
-
 // Test validateServiceFile function
 
 func TestValidateServiceFile(t *testing.T) {
 	data, _ := readServiceFile("./tests/service-valid")
-	warnings, err := validateServiceFile(data)
-	require.NoError(t, err)
+	warnings := validateServiceFile(data)
 	require.True(t, (len(warnings) == 0))
 }
 
 func TestValidateServiceFileMalFormatted(t *testing.T) {
 	data, _ := readServiceFile("./tests/service-file-mal-formatted")
-	warnings, err := validateServiceFile(data)
-	require.Error(t, err)
-	require.True(t, (len(warnings) == 0))
+	warnings := validateServiceFile(data)
+	require.Equal(t, 1, len(warnings))
 }
 
 func TestValidateServiceFileWithErrors(t *testing.T) {
 	data, _ := readServiceFile("./tests/service-file-invalid")
-	warnings, err := validateServiceFile(data)
-	require.NoError(t, err)
+	warnings := validateServiceFile(data)
 	require.Equal(t, 1, len(warnings))
 }
 
 func TestValidateServiceFileWithMultipleErrors(t *testing.T) {
 	data, _ := readServiceFile("./tests/service-multiple-errors")
-	warnings, err := validateServiceFile(data)
-	require.NoError(t, err)
-	require.Equal(t, 2, len(warnings))
+	warnings := validateServiceFile(data)
+	require.Equal(t, 1, len(warnings))
```

Comment:
there is only 1 error where this test is about multiple errors

Reply:
Good point, now it's one parsing error instead of multiple validation like before, we could get rid of this test or rename it as parsing error test